### PR TITLE
Add link to audio version of song

### PR DESCRIPTION
**Pull request by rugk**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            |  Fixes https://github.com/babel/babel/issues/8319
| Patch: Bug Fix?          | n
| Major: Breaking Change?  | n
| Minor: New Feature?      | n
| Tests Added + Pass?      | N/A
| Documentation PR Link    | n
| Any Dependency Changes?  | n
| License                  | MIT

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ For questions and support please visit join our [Slack Community](https://slack.
 
 ### Is there a Babel song?
 
-I'm so glad you asked: [Hallelujah —— In Praise of Babel](SONG.md) by [@angus-c](https://github.com/angus-c). Tweet us your recordings!
+I'm so glad you asked: [Hallelujah —— In Praise of Babel](SONG.md) by [@angus-c](https://github.com/angus-c), [audio version](https://youtu.be/40abpedBKK8) by [Swyx](https://medium.com/@swyx). Tweet us your recordings!
```
Comment (existentialism) [inline README.md]: Should we link to @swyx's twitter instead?

Comment (rugk) [reply README.md]: …or GitHub…

Comment (existentialism) [reply README.md]: That's fine too!

Comment (sw-yx) [reply README.md]: oh hi haha. sure twitter is best

**Pull request merged**